docs/layouts/shortcodes/generated/forst_configurable_configuration.html

```diff
@@ -0,0 +1,168 @@
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.compaction.style</h5></td>
+            <td style="word-wrap: break-word;">LEVEL</td>
+            <td><p>Enum</p></td>
+            <td>The specified compaction style for DB. Candidate compaction style is LEVEL, FIFO, UNIVERSAL or NONE, and Flink chooses 'LEVEL' as default style.<br /><br />Possible values:<ul><li>"LEVEL"</li><li>"UNIVERSAL"</li><li>"FIFO"</li><li>"NONE"</li></ul></td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.compression.per.level</h5></td>
+            <td style="word-wrap: break-word;">SNAPPY_COMPRESSION</td>
+            <td><p>List&lt;Enum&gt;</p></td>
+            <td>A semicolon-separated list of Compression Type. Different levels can have different compression policies. In many cases, lower levels use fast compression algorithms, while higher levels with more data use slower but more effective compression algorithms. The N th element in the List corresponds to the compression type of the level N-1When <code class="highlighter-rouge">state.backend.forst.compaction.level.use-dynamic-size</code> is true, compression_per_level[0] still determines L0, but other elements are based on the base level and may not match the level seen in the info log<br />Note: If the List size is smaller than the level number, the undefined lower level uses the last Compression Type in the List<br />Some commonly used compression algorithms for candidates include <code class="highlighter-rouge">NO_COMPRESSION</code> ,<code class="highlighter-rouge">SNAPPY_COMPRESSION</code> and <code class="highlighter-rouge">LZ4_COMPRESSION</code><br />The default value is <code class="highlighter-rouge">SNAPPY_COMPRESSION</code>, which means that all data uses the Snappy compression algorithm.Likewise, if set to <code class="highlighter-rouge">NO_COMPRESSION</code> , means that all data is not compressed, which will achieve faster speed but will bring some space amplification.In addition, if we need to consider both spatial amplification and performance, we can also set it to '<code class="highlighter-rouge">NO_COMPRESSION</code>;<code class="highlighter-rouge">NO_COMPRESSION</code>;<code class="highlighter-rouge">LZ4_COMPRESSION</code>', which means that L0 and L1 data will not be compressed, and other data will be compressed using LZ4.<br /><br />Possible values:<ul><li>"NO_COMPRESSION"</li><li>"SNAPPY_COMPRESSION"</li><li>"ZLIB_COMPRESSION"</li><li>"BZLIB2_COMPRESSION"</li><li>"LZ4_COMPRESSION"</li><li>"LZ4HC_COMPRESSION"</li><li>"XPRESS_COMPRESSION"</li><li>"ZSTD_COMPRESSION"</li><li>"DISABLE_COMPRESSION_OPTION"</li></ul></td>
```

Comment:
nits:
 type - > types
N-1When -> N-1. When
algorithm.Likewise -> algorithm. Likewise
amplification.In -? amplification. In